redirect/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "redirect-tests",
+  "version": "1.0.0",
```

Comment:
Same here; because there is no tests we don't need the boilerplate. Let's remove so there's no confusion about that.

(another option if you feel like it: add a test!)